src/style.css

```diff
@@ -1,3 +1,85 @@
-body {
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+.nav__item,
+.link {
+  text-decoration: none;
+}
+
+header {
```

Comment:
Review the [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)
<img width="1036" alt="Знімок екрана 2024-10-15 о 15 31 30" src="https://github.com/user-attachments/assets/d4562abb-fa06-4c16-a29a-9c2d89233824">

```suggestion
.header {
```